Userland/Utilities/disasm.cpp

```diff
@@ -24,15 +25,9 @@ int main(int argc, char** argv)
         "Disassemble an executable, and show human-readable "
         "assembly code for each function.");
     args_parser.add_positional_argument(path, "Path to i386 binary file", "path");
-    args_parser.parse(argc, argv);
+    args_parser.parse(args);
 
-    auto file_or_error = Core::MappedFile::map(path);
-    if (file_or_error.is_error()) {
-        warnln("Could not map file: {}", file_or_error.error());
-        return 1;
-    }
-
-    auto& file = *file_or_error.value();
+    auto& file = *TRY(Core::MappedFile::map(path));
```

Comment:
This is incorrect, you're dereferencing a temporary, which results in a use after free.